<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialSettingsEntry</class>
 <widget class="QWidget" name="SerialSettingsEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>237</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">Serial Settings Entry</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Serial Device</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="4">
       <widget class="QLabel" name="NumLEDsLabel">
        <property name="text">
         <string>Number of LEDs:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="BaudLabel">
        <property name="text">
         <string>Baud:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QComboBox" name="ProtocolComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="NameLabel">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="NameEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="ProtocolLabel">
        <property name="text">
         <string>Protocol:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="BaudEdit"/>
      </item>
      <item row="2" column="5">
       <widget class="QLineEdit" name="NumLEDsEdit"/>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="PortLabel">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QComboBox" name="PortComboBox">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::NoInsert</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>NameEdit</tabstop>
  <tabstop>BaudEdit</tabstop>
  <tabstop>NumLEDsEdit</tabstop>
  <tabstop>ProtocolComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
